package.json

```diff
@@ -72,7 +73,8 @@
       "service": true
     },
     "features": {
-      "preserve_computed": true
+      "preserve_computed": true,
+      "odata_new_parser": true
```

Comment:
It only works with the new parser? → means we can't fully release it GA yet, i.e. need to add a disclaimer à la "for early adopters, use at your own risk, ...", correct?

@johannes-vogel

Reply:
Yes, it only works in the new parser. I agree that we should add a disclaimer.

Reply:
Disclaimer added: https://github.com/SAP-samples/cap-sflight/pull/302/commits/56794dcc1a63909627cd516c8e55e752db0e7156